### DISCUSSION TITLE
Does it work with git-crypt?

### DISCUSSION THREAD

**Discussion by syrm** [gitbutlerapp/gitbutler#4107] [Q&A] [answered]
When I try gitbutler on a project with git-crypt, it shows me all the encrypted files as modified but they are not modified and it displays "Binary content not shown".

**Comment by Byron** [comment, marked as answer]
Indeed, GitButler does not currently support clean and smudge filters, an issue which [is tracked here](https://github.com/gitbutlerapp/gitbutler/issues/2854). It's also a limitation of `libgit2`, which it inherited.
To stay informed, subscribing there would probably be best.
